tests/runner.py

```diff
@@ -244,6 +244,7 @@ def create_test_file(name, contents, binary=False):
   'wasm3',
   'wasms',
   'wasmz',
+  'strict'
```

Comment:
this will run it locally by default, I think? does everything pass now?

Reply:
This change allows me to run `strict.foo` from the command line.

Without this the runner doesn't know about a suite called `strict`.

As far as I know this doesn't run strict tests unless a user asks to run it.  Is there some command line I don't know about that runs all test suites?

Reply:
I was thinking about `./tests/runner.py *.test_loop`, but looks like that does even more, so I was wrong, sorry, this looks fine.